src/app/features/project/wiki/components/wiki-list/wiki-list.component.html

```diff
@@ -0,0 +1,73 @@
+    <p-skeleton width="100%" height="1.5rem" class="mb-2 flex" />
+    <p-skeleton width="100%" height="1.5rem" class="mb-4 flex" />
+
+    <p-skeleton width="100%" height="2.5rem" class="mb-2 flex" />
+    <p-skeleton width="100%" height="1.5rem" class="mb-2 flex" />
+    <p-skeleton width="100%" height="1.5rem" class="mb-2 flex" />
+  } @else {
+    @if (expanded()) {
+      <p-panel showHeader="false">
+        <div class="flex aign-items-center justify-content-between mt-3 mb-2">
+          <p-button
+            label="{{ 'project.wiki.addNewWiki' | translate }}"
+            icon="fas fa-plus"
+            severity="success"
+            outlined
+            (click)="openAddWikiDialog()"
+          />
+          <p-button icon="fas fa-backward" severity="contrast" raised outlined (click)="collapseNavigation()">
+          </p-button>
+        </div>
+        @if (!isHomeWikiSelected() || !list().length) {
+          <p-button
+            icon="fas fa-trash"
+            [label]="'common.buttons.delete' | translate"
+            severity="danger"
+            outlined
+            (click)="openDeleteWikiDialog()"
+            class="mb-2 flex"
+          >
+          </p-button>
+        }
+        <div class="flex flex-column justify-center mt-4 flex-1">
+          <p-panelmenu [model]="wikiMenu()" [multiple]="true" class="w-full flex flex-1 flex-column">
+            <ng-template #item let-item>
+              <div class="flex align-items-center p-2 cursor-pointer" [class.active]="item.id === currentWikiId()">
+                @switch (item.type) {
+                  @case (wikiItemType.Folder) {
+                    <i [class]="item.expanded ? 'fa-minus' : 'fa-plus'" class="fas text-xs mr-1"></i>
+                    <i [class]="item.expanded ? 'fas fa-folder-open' : 'fas fa-folder'"></i>
+                    <h4 class="ml-2">{{ item.label }}</h4>
+                  }
+                  @case (wikiItemType.Component) {
+                    <i [class]="item.expanded ? 'fa-minus' : 'fa-plus'" class="fas text-xs mr-1"></i>
+                    <i class="fa-solid fa-file-lines"></i>
+                    <span class="ml-2">{{ item.label }}</span>
+                  }
+                  @default {
+                    <div>
+                      <i class="far fa-file"></i>
+                      <span class="ml-2">{{ item.label }}</span>
+                    </div>
+                  }
+                }
+              </div>
+            </ng-template>
+          </p-panelmenu>
+        </div>
+      </p-panel>
+    } @else {
+      <div class="mb-2">
+        <p-button class="btn-icon-only" (click)="collapseNavigation()" raised outlined severity="contrast">
+          <i class="fas fa-list-ul"></i>
+          <i class="fas fa-forward"></i>
+        </p-button>
+      </div>
+      <p-button icon="pi pi-plus" raised outlined severity="success" (click)="openAddWikiDialog()" />
```

Comment:
```suggestion
      <p-button icon="fas fa-plus" raised outlined severity="success" (click)="openAddWikiDialog()" />
```
Use fontawesome icon